appveyor.yml

```diff
@@ -0,0 +1,25 @@
+version: 4.1.{build}
+
+image:
+- Visual Studio 2015
+- Visual Studio 2013
+
+environment:
+  matrix:
+    - PYTHON_VER: Python27
+    - PYTHON_VER: Python27-x64
+    - PYTHON_VER: Python33
+    - PYTHON_VER: Python33-x64
+
```

Comment:
3.3 is EOL, but 3.4/3.5/3.6 is used for enterprise tasks. I expect, that it should be built

Reply:
Fair, I was just grabbing a few versions from the list they have preinstalled in order to
get a test of multibuilds going, will adjust to the actual versions once what is needed is
addressed in libyaml